src/tbb/governor.cpp

```diff
@@ -192,6 +192,22 @@ static std::uintptr_t get_stack_base(std::size_t stack_size) {
 #endif /* __TBB_USE_WINAPI */
 }
 
+static std::size_t get_stack_size(arena &a) {
+#if __linux__ && !__bg__
+    size_t np_stack_size = 0;
+    pthread_attr_t np_attr_stack;
+    if (0 == pthread_getattr_np(pthread_self(), &np_attr_stack)) {
+        if (0 == pthread_attr_getstacksize(&np_attr_stack, &np_stack_size)) {
```

Comment:
Will this call be sufficient?

Reply:
I think it is sufficient.
If stack size is not set by pthread_attr_setstacksize(), inside pthread_attr_getstacksize() can handle it. (by RLIMIT_STACK)